### ISSUE TITLE
UI: Possibility to disable certain labeling modes for a task?

### ISSUE THREAD

**Issue by niclaswue** [openvinotoolkit/cvat#4281]
Not sure, if I missed this in the documentation. My situation is the following:
I have multiple datasets to be annotated. Some using boxes and Some using tags. If a labeler is confused about the annotation type, the labels may be in an incorrect format and thus can not be merged with the dataset. At the moment, I am syncing the annotations via the API and have some backend code to filter out unexpected labels. However, in my opinion it would make much more sense to tell CVAT the allowed labeling tools during job creation. Is this possible somehow? 
Another, less preferred option to disable tools, could be a user setting which disables certain tools for all jobs the user has. In this case, I could at least tell the labelers to disable ploygon and point labeling tools.

Thanks!

**Comment by nmanovic** [comment]
@niclaswue , I agree that when a user creates a task or project, it should be possible to specify type of expected annotations to limit the amount of expected shapes (e.g. detection -> only bounding boxes are allowed). It isn't implemented yet. At the moment you can assign a link with documentation to every task which explains how your task has to be annotated: see the issue tracker field below.

![image](https://user-images.githubusercontent.com/40690625/152411531-51d7b72a-2138-4c53-9430-151133dbce73.png)

**Comment by niclaswue** [comment]
Thank you for the response. I think such a feature could be beneficial to many people, especially when labeling is outsourced. The issue tracker URL is useful, too. Didn't know about that.

**Comment by digitalpixi** [comment]
I think the Issue Tracker is very useful to address this problem, but agree it would be very useful to turn off some features depending on the Task (Draw New Polygon on, but not Polyline, Draw new rectangle etc.) Infrequently users will use Polyline and the execution looks almost like Draw New Polygon (except the shape doesn't complete/close).
